lua/lazyvim/plugins/extras/coding/codecompanion.lua

```diff
@@ -0,0 +1,47 @@
+return {
+  {
+    "olimorris/codecompanion.nvim",
+    cmd = { "CodeCompanion", "CodeCompanionActions", "CodeCompanionToggle", "CodeCompanionAdd", "CodeCompanionChat" },
+
+    opts = function()
+      local options = require("codecompanion.config")
+      local user = vim.env.USER or "User"
+
+      options.strategies.chat.roles = {
+        llm = "  CodeCompanion",
+        user = "  " .. user:sub(1, 1):upper() .. user:sub(2),
+      }
+
+      options.strategies.chat.keymaps.close.modes = {
+        n = "q",
+        i = "<C-c>",
```

Comment:
```suggestion
        i = "<Esc>",
```
Just to make sure there's no `<C-c>` left. There was another [issue at kickstart](https://github.com/nvim-lua/kickstart.nvim/issues/1121) with `<C-c>` and deferring clipboard.

Relevant [comment](https://github.com/neovim/neovim/pull/29581#issuecomment-2318966355) from Neovim developer. I really hope I'm not perceived as persistent with nitpicking. This is just to avoid potential issues that might or might not occur due to `<C-c>`.

Reply:
Sure, the issue is still happening. don't worry about being persistent on it :)